### PR TITLE
update code and add a file to enable sorting

### PR DESCRIPTION
**Pull request by AndyBeeee**

### CODE REVIEW

**Comment by NRMSMITH** [comment]
Works great - only thing is that the URL doesn't change - it would be really handy for the user if it did! Please have a look at the react notes on useSearchParams :)

File: src/Sort.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import { useSearchParams } from 'react-router-dom'
+
+function SortFunc({ setSort, setOrder }) {
+
+    const [currentSort, setCurrentSort] = useState('created_at')
+    const [currentOrder, setCurrentOrder] = useState('desc')
+    const [searchParams, setSearchParams] = useSearchParams()
+  
+    const ifClickSort = (sort) => {
```
Comment (RichardNeat) [inline src/Sort.jsx:10]: These two functions are not very DRY. Could we clean this up a bit?

File: src/Sort.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        }
+  
+    const ifClickOrder = (order) => {
+        setOrder(order)
+        setCurrentOrder(order)
+        setSearchParams({ sort: currentSort, order: order })
+        }
+
+    return (
+        <div className="sortContainer">
```
Comment (RichardNeat) [inline src/Sort.jsx:23]: Try and avoiding using divs where possible, can we think of a more semantic html tag to use here?

**Review by RichardNeat — APPROVED** [review]
Nice one Andy - this is looking really great and you have a nice flow for the user on the browser now. Just a couple of comments below for suggested tweaks, also I noticed that you have a warning on the home page that no routes match "\", you may want to add a route that exists on this url to remove that error :) Once you're happy you can go ahead and merge 👍

**Pull request merged**